projects/angular-calendar/schematics/ng-add/index.ts

```diff
@@ -156,6 +158,26 @@ function addModuleToImports(options: Schema): Rule {
       );
     }
 
+    if (options.dateAdapter === 'luxon') {
```

Comment:
Let's add some tests for this based on the moment ones:

https://github.com/angelaki/angular-calendar/blob/7740d9aa0faba3e87604abec3c5e25f3944ff6b9/projects/angular-calendar/schematics/ng-add/index.spec.ts#L59-L67

https://github.com/angelaki/angular-calendar/blob/7740d9aa0faba3e87604abec3c5e25f3944ff6b9/projects/angular-calendar/schematics/ng-add/index.spec.ts#L124-L148